src/engine/renderer/tr_local.h

```diff
@@ -1147,6 +1148,7 @@ enum class dynamicLightRenderer_t { LEGACY, TILED };
 
 		int             deformIndex;
 		bool        overrideNoPicMip; // for images that must always be full resolution
+		bool overrideFitScreen;
```

Comment:
Why do we need both `overrideFitScreen` and `fitScreen`? it sounds like we could just make do with `fitScreen`

Reply:
This was bad cargo cult reproducing something stupid already done with`shader.noPicMip` and `stage.overrideNoPicMip`, the `override` name is only meaningful for enums, not boolean, to set something an enum value is used. I'm just renaming that as `stage.fitScreen` and in the future one will have to rename `stage.overrideNoPicMip` as well.

Reply:
The reason why there are two variables is because one is for shaders (applying to all stages), one is for stages.

I'm not sure it makes sense to have it as a stage keyword though, but the same image could be used in UI (and then use fitScreen) while also being part of a 3D shader blended with other images that do not need this feature.